x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_key_flyout.tsx

```diff
@@ -251,243 +296,440 @@ export const ApiKeyFlyout: FunctionComponent<ApiKeyFlyoutProps> = ({
-                          <FormattedMessage
-                            id="xpack.security.accountManagement.apiKeyFlyout.typeLabel"
-                            defaultMessage="Type"
-                          />
-                        }
-                      >
+              <EuiPanel hasBorder>
+                {apiKey ? (
+                  <>
+                    <EuiTitle size="xs">
+                      <h4>
+                        <FormattedMessage
+                          id="xpack.security.accountManagement.apiKeyFlyout.overviewLabel"
+                          defaultMessage="Overview"
+                        />
+                      </h4>
+                    </EuiTitle>
+                    <EuiSpacer />
+                    <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+                      <EuiFlexItem>
+                        <EuiText size="xs">
+                          <strong>
+                            <FormattedMessage
+                              id="xpack.security.accountManagement.apiKeyFlyout.nameLabel"
+                              defaultMessage="Name"
+                            />
+                          </strong>
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>{apiKey.name}</EuiFlexItem>
+                    </EuiFlexGroup>
+                    <EuiHorizontalRule margin="s" />
+                    <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+                      <EuiFlexItem>
+                        <EuiText size="xs">
+                          <strong>
+                            <FormattedMessage
+                              id="xpack.security.accountManagement.apiKeyFlyout.ownerLabel"
+                              defaultMessage="Owner"
+                            />
+                          </strong>
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <EuiFlexGroup justifyContent="flexEnd" alignItems="center" gutterSize="xs">
+                          <EuiFlexItem grow={false}>
+                            <EuiIcon type="user" />
+                          </EuiFlexItem>
+                          <EuiFlexItem grow={false}>{apiKey.username}</EuiFlexItem>
+                        </EuiFlexGroup>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                    <EuiHorizontalRule margin="s" />
+
+                    <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+                      <EuiFlexItem>
+                        <EuiText size="xs">
+                          <strong>
+                            <FormattedMessage
+                              id="xpack.security.accountManagement.apiKeyFlyout.typeLabel"
+                              defaultMessage="Type"
+                            />
```

Comment:
Nit: should use the const `TypeLabel` from below.

Reply:
Done